src/Core/Usecase/Post/WebhookUpdatePost.php

```diff
@@ -35,17 +35,11 @@ public function interact()
         // ... persist the changes
         $this->repo->updateFromService($entity);
 
-        // ... check that the entity can be read by the current user
-        if ($this->auth->isAllowed($entity, 'read')) {
```

Comment:
@willdoran this check was always failing, because we never have a user session on this request. Is the desired result to always return an empty 204 response? or to always return a 200 with the updated post? I've gone with 200, but I suspect it was previously a 204

Reply:
A 200 is fine, now that we have lumen the service proxy will hopefully go away completely and live inside Core. Right now it does nothing with the response unless it fails in which case it will retry 3 times.